From: =?utf-8?q?David_Pr=C3=A9vot?= <taffit@debian.org>
Date: Thu, 13 Nov 2014 21:39:53 -0400
Subject: Update POT file

Some strings positions (lines in files) changed, thus the comment
changes, but there is no actual (functional) change. The purpose is
only to ensure the .gmo files get rebuild.
---
 po/gnupg.pot | 58 +++++++++++++++++++++++++++++-----------------------------
 1 file changed, 29 insertions(+), 29 deletions(-)

diff --git a/po/gnupg.pot b/po/gnupg.pot
index f346f5e..2e8d049 100644
--- a/po/gnupg.pot
+++ b/po/gnupg.pot
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: GNU gnupg 1.4.18\n"
 "Report-Msgid-Bugs-To: gnupg-i18n@gnupg.org\n"
-"POT-Creation-Date: 2014-06-30 19:59+0200\n"
+"POT-Creation-Date: 2014-11-13 21:39-0400\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
@@ -823,12 +823,12 @@ msgid "--output doesn't work for this command\n"
 msgstr ""
 
 #: g10/delkey.c:73 g10/export.c:309 g10/keyedit.c:1537 g10/keyedit.c:3392
-#: g10/keyserver.c:1821 g10/revoke.c:226
+#: g10/keyserver.c:1837 g10/revoke.c:226
 #, c-format
 msgid "key \"%s\" not found: %s\n"
 msgstr ""
 
-#: g10/delkey.c:81 g10/export.c:339 g10/import.c:2487 g10/keyserver.c:1835
+#: g10/delkey.c:81 g10/export.c:339 g10/import.c:2487 g10/keyserver.c:1851
 #: g10/revoke.c:232 g10/revoke.c:476
 #, c-format
 msgid "error reading keyblock: %s\n"
@@ -4159,7 +4159,7 @@ msgstr ""
 msgid "Enter number(s), N)ext, or Q)uit > "
 msgstr ""
 
-#: g10/keyserver.c:823 g10/keyserver.c:1535
+#: g10/keyserver.c:823 g10/keyserver.c:1551
 #, c-format
 msgid "invalid keyserver protocol (us %d!=handler %d)\n"
 msgstr ""
@@ -4173,121 +4173,121 @@ msgstr ""
 msgid "key not found on keyserver\n"
 msgstr ""
 
-#: g10/keyserver.c:1229
+#: g10/keyserver.c:1245
 #, c-format
 msgid "requesting key %s from %s server %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1233
+#: g10/keyserver.c:1249
 #, c-format
 msgid "requesting key %s from %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1257
+#: g10/keyserver.c:1273
 #, c-format
 msgid "searching for names from %s server %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1260
+#: g10/keyserver.c:1276
 #, c-format
 msgid "searching for names from %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1438
+#: g10/keyserver.c:1454
 #, c-format
 msgid "sending key %s to %s server %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1442
+#: g10/keyserver.c:1458
 #, c-format
 msgid "sending key %s to %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1485
+#: g10/keyserver.c:1501
 #, c-format
 msgid "searching for \"%s\" from %s server %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1488
+#: g10/keyserver.c:1504
 #, c-format
 msgid "searching for \"%s\" from %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1495 g10/keyserver.c:1598
+#: g10/keyserver.c:1511 g10/keyserver.c:1614
 msgid "no keyserver action!\n"
 msgstr ""
 
-#: g10/keyserver.c:1543
+#: g10/keyserver.c:1559
 #, c-format
 msgid "WARNING: keyserver handler from a different version of GnuPG (%s)\n"
 msgstr ""
 
-#: g10/keyserver.c:1552
+#: g10/keyserver.c:1568
 msgid "keyserver did not send VERSION\n"
 msgstr ""
 
-#: g10/keyserver.c:1621 g10/keyserver.c:2144
+#: g10/keyserver.c:1637 g10/keyserver.c:2160
 msgid "no keyserver known (use option --keyserver)\n"
 msgstr ""
 
-#: g10/keyserver.c:1627
+#: g10/keyserver.c:1643
 msgid "external keyserver calls are not supported in this build\n"
 msgstr ""
 
-#: g10/keyserver.c:1639
+#: g10/keyserver.c:1655
 #, c-format
 msgid "no handler for keyserver scheme `%s'\n"
 msgstr ""
 
-#: g10/keyserver.c:1644
+#: g10/keyserver.c:1660
 #, c-format
 msgid "action `%s' not supported with keyserver scheme `%s'\n"
 msgstr ""
 
-#: g10/keyserver.c:1652
+#: g10/keyserver.c:1668
 #, c-format
 msgid "%s does not support handler version %d\n"
 msgstr ""
 
-#: g10/keyserver.c:1659
+#: g10/keyserver.c:1675
 msgid "keyserver timed out\n"
 msgstr ""
 
-#: g10/keyserver.c:1664
+#: g10/keyserver.c:1680
 msgid "keyserver internal error\n"
 msgstr ""
 
-#: g10/keyserver.c:1673
+#: g10/keyserver.c:1689
 #, c-format
 msgid "keyserver communications error: %s\n"
 msgstr ""
 
-#: g10/keyserver.c:1698 g10/keyserver.c:1732
+#: g10/keyserver.c:1714 g10/keyserver.c:1748
 #, c-format
 msgid "\"%s\" not a key ID: skipping\n"
 msgstr ""
 
-#: g10/keyserver.c:1991
+#: g10/keyserver.c:2007
 #, c-format
 msgid "WARNING: unable to refresh key %s via %s: %s\n"
 msgstr ""
 
-#: g10/keyserver.c:2013
+#: g10/keyserver.c:2029
 #, c-format
 msgid "refreshing 1 key from %s\n"
 msgstr ""
 
-#: g10/keyserver.c:2015
+#: g10/keyserver.c:2031
 #, c-format
 msgid "refreshing %d keys from %s\n"
 msgstr ""
 
-#: g10/keyserver.c:2071
+#: g10/keyserver.c:2087
 #, c-format
 msgid "WARNING: unable to fetch URI %s: %s\n"
 msgstr ""
 
-#: g10/keyserver.c:2077
+#: g10/keyserver.c:2093
 #, c-format
 msgid "WARNING: unable to parse URI %s\n"
 msgstr ""
